opentelemetry-quickstart/src/main/java/org/acme/opentelemetry/TracedResource.java

```diff
@@ -18,6 +24,23 @@ public class TracedResource {
     @Context
     UriInfo uriInfo;
 
+    @Inject
+    OpenTelemetry openTelemetry;
+
+    private Meter meter;
```

Comment:
you can inject the Meter with 
```
@Inject
Meter meter;
```

Reply:
I'd prefer to keep it as it is because currently Meter is obtained from OpenTelemetry and it works. If I do that `@Inject Meter meter`, then I will have `UnsatisfiedResolutionException`  CDI cannot find the bean to inject...and I am not sure if I will need to refactor a lot of the current code removing `@PostConstruct init` , etc...

Reply:
It has to work. Please make sure you are injecting `io.opentelemetry.api.metrics.Meter`

Reply:
As I mentioned, this `io.opentelemetry.api.metrics.Meter` injection led to:
```
[ERROR] Errors: 
[ERROR]   TracedResourceTest.testChainEndpoint » Runtime java.lang.RuntimeException: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
        [error]: Build step io.quarkus.arc.deployment.ArcProcessor#validate threw an exception: jakarta.enterprise.inject.spi.DeploymentException: jakarta.enterprise.inject.UnsatisfiedResolutionException: Unsatisfied dependency for type io.opentelemetry.api.metrics.Meter and qualifiers [@Default]
        - injection target: org.acme.opentelemetry.TracedResource#meter
        - declared on CLASS bean [types=[org.acme.opentelemetry.TracedResource, java.lang.Object], qualifiers=[@Default, @Any], target=org.acme.opentelemetry.TracedResource]
        at io.quarkus.arc.processor.BeanDeployment.processErrors(BeanDeployment.java:1569)
        at io.quarkus.arc.processor.BeanDeployment.init(BeanDeployment.java:338)
        at io.quarkus.arc.processor.BeanProcessor.initialize(BeanProcessor.java:167)
        at io.quarkus.arc.deployment.ArcProcessor.validate(ArcProcessor.java:490)
        at java.base/java.lang.invoke.MethodHandle.invokeWithArguments(MethodHandle.java:733)
        at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:856)
        at io.quarkus.builder.BuildContext.run(BuildContext.java:256)
        at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
        at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2675)
        at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2654)
        at org.jboss.threads.EnhancedQueueExecutor.runThreadBody(EnhancedQueueExecutor.java:1627)
        at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1594)
        at java.base/java.lang.Thread.run(Thread.java:1583)
        at org.jboss.threads.JBossThread.run(JBossThread.java:499)
Caused by: jakarta.enterprise.inject.UnsatisfiedResolutionException: Unsatisfied dependency for type io.opentelemetry.api.metrics.Meter and qualifiers [@Default]
        - injection target: org.acme.opentelemetry.TracedResource#meter
        - declared on CLASS bean [types=[org.acme.opentelemetry.TracedResource, java.lang.Object], qualifiers=[@Default, @Any], target=org.acme.opentelemetry.TracedResource]
        at io.quarkus.arc.processor.Beans.resolveInjectionPoint(Beans.java:546)
        at io.quarkus.arc.processor.BeanInfo.init(BeanInfo.java:698)
        at io.quarkus.arc.processor.BeanDeployment.init(BeanDeployment.java:323)
        ... 12 more
```

Reply:
It's because of the property names